package.json

```
@@ -50,19 +50,19 @@
     "@typescript-eslint/parser": "^4.21.0",
     "cpy-cli": "^3.1.1",
     "eslint": "^7.24.0",
-    "eslint-config-prettier": "^8.1.0",
-    "eslint-config-standard": "^16.0.2",
-    "eslint-plugin-import": "^2.22.1",
-    "eslint-plugin-jest": "^24.3.4",
+    "eslint-config-kentcdodds": "^18.0.1",
```

https://github.com/kentcdodds/eslint-config-kentcdodds/releases/tag/v18.1.0

```suggestion
    "eslint-config-kentcdodds": "^18.1.0,
```